keyboards/misterknife/knife66/readme.md

```diff
@@ -0,0 +1,18 @@
+# Knife66
+
+![Knife66](https://i.imgur.com/p7G2g81.png)
```

Comment:
Swaps the image here for a smaller version.

```suggestion
![Knife66](https://i.imgur.com/p7G2g81l.png)
```

Reply:
done